manual_user_install.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+
+# tested with a clean 64 bit installation of Raspbian-Lite
+
+function install_core (){
+    echo "Installing OVOS core"
+    echo
+    pip3 install -U pip
+
+    # install ovos-core - ovos-backend-client has conflicting dependencies
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-backend-client
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-core
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-audio
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ocp-audio-plugin
+    pip3 install tornado
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-messagebus
+    pip3 install adapt-parser
+    pip3 install padacioso
+
+    # dinkum listener
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-dinkum-listener
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-vad-plugin-webrtcvad
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-vad-plugin-silero
+
+    #Precise-lite wake-word cluster
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-precise-lite
+    python3 -m pip3 install --upgrade setuptools
+    # if pip3 install tflite_runtime is problematic try:
+    # we install full tensorflow (because it just works, takes a while to install)
+    # sudo apt install -y libatlas-base-dev
+    #  pip3 install tensorflow --no-cache-dir
+    pip3 install tflite_runtime
+    pip3 install PyYAML
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-workshop
+    sudo apt install -y python3-fann2
+    pip3 install padatious
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-lingua-franca
+    pip3 install PyAudio
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-pocketsphinx
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-precise
+    # pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-vosk
+
+    # install speech to text
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-stt-plugin-server
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-stt-plugin-selene
+    # pip3 install git+https://github.com/OpenVoiceOS/ovos-stt-plugin-vosk
+
+    # install text to speech
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-microphone-plugin-alsa
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-plugin-mimic3-server
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-server-plugin
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-plugin-mimic
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-plugin-mimic2
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-plugin-google-tx
```

Comment:
i would only include mimic3 server + piper plugin

Reply:
removed server-plugin, plugin-mimic, plugin-mimic2, and google-tx.  Added mimic3-server and plugin-piper.  Will test.

Reply:
for piper you will also need sudo apt-get install espeak-ng

maybe also leave ovos-tts-server-plugin, as that brings automatic support for most remote plugins and will soon replace mimic3  (once we have public servers list for piper)

Reply:
Done..

Reply:
it works